### PR TITLE
Em 19 - Add Endpoint Delete Appointments

### PR DESCRIPTION
**Pull request by MiguelAngelSF**

### CODE REVIEW

**Review by gfranco-arkus — APPROVED** [review]

**Review by Navi02643 — APPROVED** [review]

File: server/services/appointments.js [hunk 1]
```diff
@@ -64,3 +64,20 @@ async function createAppointment(appointment) {
 }
 
 module.exports = { getAppointments, createAppointment };
+async function deleteAppointment(appointment, range) {
+  const date = new Date();
+  const data = { _id: appointment._id };
+  const search = await appointmentDB.searchAppointment(data);
+  if (range === 1) {
```
Comment (ErickIsra01) [inline server/services/appointments.js:71]: I think this nested if could be changed into an if with two conditions

File: server/database/users.js [hunk 2]
```diff
@@ -11,9 +11,16 @@ async function findEmail(email) {
   return searchEmail;
 }
 
+async function findPatient(name) {
+  const searchPatient = await UserModel.find({ name: { $regex: `${name}` }, range: 1 });
```
Comment (FraGa96) [inline server/database/users.js:15]: use constants/enums instead of the plain number in range (check comments on PR #21 )

**Review by FraGa96 — CHANGES_REQUESTED** [review]

File: server/services/appointments.js [hunk 3]
```diff
@@ -3,6 +3,12 @@ const tokenDB = require("../database/tokens");
 const appointmentDB = require("../database/appointments");
 const appointmentDTO = require("./models/appointmentsDTO");
 
+const RANGE = {
```
Comment (FraGa96) [inline server/services/appointments.js]: move this to a different file where you can have all constants, I'm pretty sure you're going to require this constant in other files

**Review by FraGa96 — CHANGES_REQUESTED** [review]

File: server/services/appointments.js [hunk 4]
```diff
@@ -2,12 +2,7 @@ const userDB = require("../database/users");
 const tokenDB = require("../database/tokens");
 const appointmentDB = require("../database/appointments");
 const appointmentDTO = require("./models/appointmentsDTO");
-
-const RANGE = {
-  patient: 1,
-  therapist: 2,
-  administrador: 3,
-};
+const ranges = require("../utils/range.constans");
```
Comment (FraGa96) [inline server/services/appointments.js]: UPPER_CASE please

**Review by FraGa96 — CHANGES_REQUESTED** [review]

**Review by FraGa96 — APPROVED** [review]

**Pull request merged**